test/e2e/serial/tests/workload_placement.go

```diff
@@ -398,7 +402,7 @@ var _ = Describe("[serial][disruptive][scheduler] numaresources workload placeme
 			Expect(err).ToNot(HaveOccurred())
 
 			// the NRT updaters MAY be slow to react for a number of reasons including factors out of our control
-			// (kubelet, runtime). This is a known behaviour. We can only tolerate some delay in reporting on pod removal.
+			// (kubelet, runtime). This is a known behavior. We can only tolerate some delay in reporting on pod removal.
```

Comment:
we really dislike the english spelling, don't we? :)

Reply:
I wouldn't mind much but seeing a line under it irritates me :)